### DISCUSSION TITLE
Ask your questions and concerns.

### DISCUSSION THREAD

**Discussion by itsmeshibintmz** [itsmeshibintmz/Acer-Nitro-5-AN515-54-Monterey#9] [Q&A] [answered]
We'll try to find solutions to your problems together.

**Comment by vaibhavkuks** [comment]
Do you plan ahead to create an EFI update for MacOS Ventura also???

**Reply by vaibhavkuks** [reply]
👉😊👈

**Reply by itsmeshibintmz** [reply]
'll update here after the stable version is released.

**Comment by itsmeshibintmz** [comment, marked as answer]
I'm planning to create an EFI for Ventura also, but most of the features Apple introduced are optimized for Apple silicon like stage manager is showing terrible performance with Intel(as per beta test usage reports). WIll look into that.

**Comment by itsmeshibintmz** [comment]
Sold my Nitro, so can't support anymore. Hope someone will provide future EFI's for our Nitro Series.
